en/docs/learn/login-to-magento-using-wso2-identity-server.md

```diff
@@ -0,0 +1,148 @@
+
+2. You’ll be redirected to the edit page of the newly created service provider. Expand the 
+**Inbound Authentication Configuration** section and then **SAML2 Web SSO Configuration** and click on 
+**Configure** to add the SAML related settings.
+
+3. We can extract the values to be filled in this section from the MiniOrange SAML SSO plugin itself. Log in to 
+your Magento admin page and click on the **Service Provider Metadata** section of the SAML plugin. You should see 
+the configuration values such as issues and ACS URL there.
+![magento-saml-sso-sp-metadata](../assets/img/tutorials/magento-saml-sso-sp-metadata.png)
+
+4. Now go back to Identity Server SP settings and add the following values, which were extracted from the above page.
+	1. Issuer
+    2. Assertion Consumer URL
+    3. Untick **Enable Signature Validation in Authentication Requests and Logout Requests** for the moment.
+
+5. Once completed, the settings page would look like below
+![magento-saml-sso-sp-configurations](../assets/img/tutorials/magento-saml-sso-sp-configurations.png)
+
+6. Click on the **Download IDP Metadata** button to download the metadata XML file, which we will need when 
+configuring the SAML SSO plugin.
+
+7. Once all done, click on the **Update** button at the bottom to save.
+
+We have one more configuration left to do on the Identity Server side. Magento expects an email as the user 
+identifier and we need to configure our service provider to send the email address as the subject on successful 
+authentication, instead of the username. Expand the **Claim Configuration** section and set the 
+**Subject Claim URI** to _**http://wso2.org/claims/emailaddress**_. Don’t forget to click on **Update** to save the configuration.
+
+## Configuring MiniOrange SAML SSO Plugin
+
+Login to Magento admin portal and click on the **Service Provider Setup** option of the SAML plugin. We need 
+to provide the following values on this page to complete the setup.
+
+1. Identity Provider Name: WSO2
+	- This will be used in your Magento login page in the SSO login button, with the text **Login with xxx**
+2. IdP Entity ID or Issuer: localhost
+3. SAML Login URL: https://localhost:9443/samlsso
+4. X.509 Certificate
+	- You can extract the certificate from the metadata XML file we downloaded earlier. Make sure to paste 
+	the certificate content in the middle of **—–BEGIN CERTIFICATE—–** and **—–END CERTIFICATE—–** tags, as instructed.
+
+Once completed, the settings page should look like below.
+![magento-saml-sso-magent-sp-settings](../assets/img/tutorials/magento-saml-sso-magent-sp-settings.png)
+
+Click on the **Test configuration** button. This will load the SAML authentication on a new browser window 
+and you should see the WSO2 Login page as a result. Once login there, you’ll be redirected back to Magento and a 
+**Test Successful** screen should appear.
+	![magento-saml-sso-test-success](../assets/img/tutorials/magento-saml-sso-test-success.png)
```

Comment:
Check alignment of the image (Should move left I guess)

![magento-saml-sso-test-success](../assets/img/tutorials/magento-saml-sso-test-success.png)